mmpose/codecs/mono_pose_lifting.py

```diff
@@ -0,0 +1,206 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+from copy import deepcopy
+from typing import Optional, Tuple
+
+import numpy as np
+
+from mmpose.registry import KEYPOINT_CODECS
+from .base import BaseKeypointCodec
+from .utils import camera_to_image_coord
+
+
+@KEYPOINT_CODECS.register_module()
+class MonoPoseLifting(BaseKeypointCodec):
+    r"""Generate keypoint coordinates for pose lifter.
```

Comment:
This docstring is the same as `VideoPoseLIfting`. Could you please refine it with more details?